tools/odbc/descriptor.cpp

```diff
@@ -857,7 +826,7 @@ SQLRETURN DescRecord::SetSqlDataType(SQLSMALLINT type) {
 		sql_desc_concise_type = type;
 		auto interval_code = OdbcInterval::GetIntervalCode(type);
 		if (interval_code == SQL_ERROR) {
-			return SQL_ERROR;
+			return SQL_ERROR; // handled
```

Comment:
What do you mean by `handled`?

Reply:
This is just a way for me to keep track of which errors I've looked at